pkg/crc/logging/logging.go

```diff
@@ -35,6 +35,10 @@ func CloseLogging() {
 	logrus.StandardLogger().ReplaceHooks(make(logrus.LevelHooks))
 }
 
+func RemoveLogFile(path string) error {
```

Comment:
It would be best to not leak the log path out of this package. But this is a larger refactoring to do.

A rule I like: when you have a constant, use it only once (or at least as few as possible, only in the same package).

Reply:
Yes, it feels like a better API for the `logging` package would remove the need for most of the complications in this patch series (additional logfile argument to `runDelete`, `NewClientWithLogFile`, ...

Seems that having `logging.Init("/some/logfile")`, `logging.RemoveLogFile()` and `logging.Close()` would be enough? 
Looking at the code of the `logging` package, it seems `InitLogrus()` will leak a file descriptor each time it's called :-/ There's a global `logfile` variable, but `InitLogrus` initializes a different `logFile` variable.